core/storage/trie/impl/trie_error.hpp

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_TRIE_ERROR_HPP
+#define KAGOME_TRIE_ERROR_HPP
+
+#include <outcome/outcome.hpp>
+
+namespace kagome::storage::trie {
+  /**
+   * @brief TrieDbError enum provides error codes for TrieDb methods
+   */
+  enum class TrieError {
+    SUCCESS = 0,   // 0 should not represent an error
```

Comment:
```suggestion
```

Reply:
This is here because 0 should not represent error

Reply:
0 should not represent an error anyway, why explicitly put it in the enum. Aside from verbosity, it is weird to have a SUCCESS entry in the error list and might lead to error messages "Error: Success" which are very confusing.